server/index.js

```diff
@@ -80,6 +80,13 @@ export default class Server {
       const p = join(__dirname, '..', 'client', ...(params.path || []))
       await serveStatic(req, res, p)
     })
+
+    this.router.get('/_webpack/:number', async (req, res, params) => {
```

Comment:
This path seems to conflict with https://github.com/zeit/next.js/blob/master/server/build/webpack.js#L179